find_intersecting_spherical_cones.py

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env python
```

Comment:
Need to add header featuring lines
Author:
Reproduced By:
Date:
- Description -